etl/paths.py

```diff
@@ -10,11 +10,13 @@
 STEP_DIR = BASE_DIR / "etl" / "steps"
 
 # Regions paths
-LATEST_REGIONS_REGIONS_YML = sorted((STEP_DIR / "data/garden/regions/").glob("*/regions.yml"))[-1]
-LATEST_REGIONS_DATASET_PATH = BASE_DIR / LATEST_REGIONS_REGIONS_YML.relative_to(STEP_DIR).with_suffix("")
+LATEST_REGIONS_VERSION = sorted((STEP_DIR / "data/garden/regions/").glob("*/regions.yml"))[-1].parts[-2]
+LATEST_REGIONS_YML = STEP_DIR / "data/garden/regions" / LATEST_REGIONS_VERSION / "regions.yml"
+LATEST_REGIONS_DATASET_PATH = BASE_DIR / "data/garden/regions" / LATEST_REGIONS_VERSION / "regions"
 
 # WB Income
-LATEST_INCOME_DATASET_PATH = BASE_DIR / LATEST_REGIONS_REGIONS_YML.relative_to(STEP_DIR).with_suffix("")
+LATEST_INCOME_VERSION = sorted((STEP_DIR / "data/garden/wb/").glob("*/wb_income.ipynb"))[-1].parts[-2]
```

Comment:
Now that we are here, we could try to use the latest income groups dataset, which is called `income_groups`, and stop using `wb_income`. However, within `key_indicators` we should keep using the old `wb_income` dataset, to avoid abrupt changes.